bundles/org.openhab.binding.dlinksmarthome/README.md

```diff
@@ -40,6 +42,13 @@ In the Thing file, this looks like e.g.
 
 - **motion** - Triggered when the sensor detects motion.
 
+### DSP-W215
+
+- **current_consumption** - the current power consumption.
```

Comment:
In general for things a table is used to show the channels. Especially with more channels that makes it a bit cleaner. You also might include the type of channel so users know what to use when creating a channel in files. You can use http://markdowntable.com/ to format the table to be evenly outlines.